refact-agent/gui/src/services/refact/chat.ts

```diff
@@ -216,10 +214,10 @@ export async function generateChatTitle({
 }: GetChatTitleArgs): Promise<Response> {
   const body = JSON.stringify({
     messages,
-    model: model,
+    model: "gpt-4o-mini",
```

Comment:
I'm wondering if there's cases where mini isn't available
if model is empty will it just use the default?